packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/tests/reducer.test.js

```diff
@@ -539,112 +727,82 @@ describe('CONTENT MANAGER | COMPONENTS | EditViewDataManagerProvider | reducer',
       ).toStrictEqual({
         ...initialState,
         initialData: {
-          relation: { results: [{ id: 2 }, { id: 1 }] },
+          relation: [{ id: 2 }, { id: 1 }],
         },
         modifiedData: {
-          relation: {
-            connect: [],
-            disconnect: [],
-          },
+          relation: [{ id: 2 }, { id: 1 }],
```

Comment:
I'm wondering: does that mean for a user (plugin-developer) it will be impossible to differentiate which relations have changed, because `initialData` and `modifiedData` are always the same? That would be a breaking change imo and defeat the idea of `modifiedData`. At the moment this is the only integration point we give to plugin developers if they want to interfact with relational fields in the UI and therefore I think we shouldn't break it.

Reply:
No `modifedData` and `initialData` will still be different, this is also my problem with the naming, in reality `initialData` is the state of the server. Whilst `modifiedData` is it the state of the browser.

The flow would go like:
* init the edit view
* ID & MD are equal
* User makes an update
* MD is updated to reflect this
* User submits entry
* `cleanData` diffs ID & MD